### PR TITLE
Fixed ambiguous usage of DnnFilePickerUploader

### PR DESCRIPTION
**Pull request by stetard**
### Description of PR...
Updates Dnn dependencies to v9.10.2 and Newtonsoft package from v10 to V13

## Changes made
- Update Nuget packages
- Fixed ambiguous usage of DnnFilePickerUploader

## PR Template Checklist

- [ ] Fixes Bug
- [ ] Feature solution
- [x] Other

## Please mark which issue is solved

Close #

### CODE REVIEW

File: DotNetNuke.Announcements.csproj [hunk 1]
```diff
@@ -280,41 +282,55 @@
...
-      <HintPath>packages\DotNetNuke.Web.9.4.0\lib\net45\DotNetNuke.WebUtility.dll</HintPath>
+    <Reference Include="DotNetNuke.Web.Client, Version=9.6.2.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>packages\DotNetNuke.Web.Client.9.6.2\lib\net45\DotNetNuke.Web.Client.dll</HintPath>
       <SpecificVersion>False</SpecificVersion>
       <Private>False</Private>
     </Reference>
-    <Reference Include="Microsoft.ApplicationBlocks.Data, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>packages\DotNetNuke.Core.9.4.0\lib\net45\Microsoft.ApplicationBlocks.Data.dll</HintPath>
+    <Reference Include="DotNetNuke.Web.Deprecated, Version=9.6.2.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>packages\DotNetNuke.Web.Deprecated.9.6.2\lib\net45\DotNetNuke.Web.Deprecated.dll</HintPath>
```
Comment (valadas) [inline DotNetNuke.Announcements.csproj:316]: This is still an issue, we don't want to bing back this deprecated package

File: DotNetNuke.Announcements.csproj [hunk 2]
```diff
@@ -337,13 +353,18 @@
...
-    <Reference Include="System.Web.Http, Version=5.2.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>packages\Microsoft.AspNet.WebApi.Core.5.2.3\lib\net45\System.Web.Http.dll</HintPath>
+    <Reference Include="System.Web.Http, Version=5.2.7.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>packages\Microsoft.AspNet.WebApi.Core.5.2.7\lib\net45\System.Web.Http.dll</HintPath>
       <SpecificVersion>False</SpecificVersion>
       <Private>False</Private>
     </Reference>
     <Reference Include="System.XML" />
     <Reference Include="System.Xml.Linq" />
+    <Reference Include="Telerik.Web.UI, Version=2013.2.717.40, Culture=neutral, PublicKeyToken=121fae78165ba3d4, processorArchitecture=MSIL">
```
Comment (valadas) [inline DotNetNuke.Announcements.csproj:363]: And we don't want to bring telerik back

File: packages.config [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
-  <package id="DotNetNuke.Web.Deprecated" version="9.4.0" targetFramework="net452" />
-  <package id="DotNetNuke.WebApi" version="9.4.0" targetFramework="net452" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net452" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net452" />
+  <package id="DotNetNuke.Core" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.DependencyInjection" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.Instrumentation" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.Web" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.Web.Client" version="9.6.2" targetFramework="net472" />
+  <package id="DotNetNuke.Web.Deprecated" version="9.6.2" targetFramework="net472" />
```
Comment (valadas) [inline packages.config:8]: I am surprised this was was still there, in a perfect world it would be nice to remove it.

**Review by valadas — CHANGES_REQUESTED** [review]

File: DotNetNuke.Announcements.csproj (hunk 1 above)
Comment (stetard) [reply DotNetNuke.Announcements.csproj:316]: I think I updated the DNN Core nugget package and it added the deprecated package including Telerik.
Maybe I can uninstall only the deprecated package to remove them.

Comment (valadas) [reply DotNetNuke.Announcements.csproj:316]: Must be something else, the core package does not depend on anything but DotNetNuke.DependencyInjection:
image

**Review by valadas — APPROVED** [review]
Excellent, now that looks good to me.

**Pull request merged**